src/Pages/hokyunkim/Main/Main.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import {withRouter} from 'react-router-dom'
+import './Main.scss'
+
+class Main extends React.Component {
+    constructor() {
+        super();
+        this.state = {
+            commentList: [],
+            commentValue: "", 
+        };
+    }
+
+
+    handleCommentValue = (event) => {
+        this.setState(
+            {
+            commentValue: event.target.value,
+            
+        })
+    };
+
+    handleBtn = (event) => {
+        this.setState({
+            commentList: [
+                ...this.state.commentList, 
+                {
+                    userName: "리액트 너무 어려워 나는 포기할래:",
+                    content: this.state.commentValue 
+                }
```

Comment:
데이터에 id도 추가해주세요! 그래야 그 댓글을 특정할 수 있어서 나중에 삭제도 할 수 있습니다!
그리고 각 데이터는 고유한 id를 가지고 있어야 합니다! 아래는 id를 주는 방법 중 하나를 예시로 넣었어요!
리액트 포기 노노!
```suggestion
                {
                   id: this.state.commentList.length,
                    userName: "리액트 충분히 가능 ;) 포기 용납 안됩니다.",
                    content: this.state.commentValue 
                }
```

Reply:
네 포기안하겠습니다! 감사합니다!